tutorials/snowplow-cli-mcp/advanced-usage.md

```diff
@@ -0,0 +1,69 @@
+---
+position: 4
+title: Advanced usage and AI analysis
+---
+
+# Advanced usage and AI analysis
+
```

Comment:
1-2 sentences intro to say why this is advanced compared to the previous page instructions